controlplane/kubeadm/internal/etcd/etcd.go

```diff
@@ -41,7 +41,7 @@ type etcd interface {
 	AlarmList(ctx context.Context) (*clientv3.AlarmResponse, error)
 	Close() error
 	Endpoints() []string
-	MemberList(ctx context.Context) (*clientv3.MemberListResponse, error)
+	MemberList(ctx context.Context, opts ...clientv3.OpOption) (*clientv3.MemberListResponse, error)
```

Comment:
@fabriziopandini We should check if we want to specifcy one of the options
/hold